jplag/src/main/java/de/jplag/options/JPlagOptions.java

```diff
@@ -80,9 +80,14 @@
     private Set<String> excludedFiles = Collections.emptySet();
 
     /**
-     * Directory that contains all submissions.
+     * Directories that contain all submissions that need to be checked for plagiarism.
      */
-    private List<String> rootDirectoryNames;
+    private List<String> plagiarismCheckRootDirectoryNames;
+
+    /**
+     * Directories that contain all submissions with prior art to check against.
+     */
+    private List<String> priorSubmissionsRootDirectoryNames;
```

Comment:
As commented above, I do not think the naming is ideal. Maybe let's do:

- submissionDirectories
- priorSubmissionDirectories/oldSubmissionDirectories

And then just adapt the java doc that it states that these lists contain the names of these directories.